Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6847,6 +6913,21 @@ WebAuthenticationEnabled:
     WebCore:
       default: true
 
+WebCodecsAV1Enabled:
+  type: bool
+  status: preview
```

Comment:
@brentfulgham: This was newly-introduced in b9c9ce85. I've given it "preview" to align with `WebCodecsEnabled` and because it was added as an experimental feature, but what do you think?

Reply:
cc @youennf to advise whether this flag is suitable for being exposed to web developers. If not, we can change it to "unstable" so that marked as in-development it's only visible in internal builds.